src/foobar/urls.py

```diff
@@ -31,6 +32,6 @@
         import debug_toolbar
         urlpatterns += [
             url(r'^__debug__/', include(debug_toolbar.urls)),
-        ]
+        ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
We should place `static` directly after the urlpatterns above. As you have mentioned, `static` has no effect when `DEBUG=True` anways, but the main reason for moving it up is that the static routes will not be applied if importing `debug_toolbar` raises an exception.